source/_docs/z-wave/device-specific.markdown

```diff
@@ -68,37 +70,10 @@ Reboot your Pi 4 without the Razberry Z-Wave hat first. Then shutdown, add the h
 #### Raspberry Pi 3 procedure
 
 ```text
-dtoverlay=pi3-disable-bt
-```
-
-Then disable the Bluetooth modem service:
-
-```bash
-sudo systemctl disable hciuart
-```
-
-Once Bluetooth is off, enable the serial interface via the `raspi-config` tool. After reboot run:
-
-```bash
-sudo systemctl mask serial-getty@ttyAMA0.service
-```
-
-so that your serial interface looks like:
-
-```text
-crw-rw---- 1 root dialout 204, 64 Sep  2 14:38 /dev/ttyAMA0
-```
-at this point simply add your user (homeassistant) to the dialout group:
-
-```bash
-sudo usermod -a -G dialout homeassistant
+dtoverlay=disable-bt
 ```
 
-<div class='note'>
-
-  If you've installed the Z-Way software, you'll need to ensure you disable it before you install Home Assistant or you won't be able to access the board. Do this with `sudo /etc/init.d/z-way-server stop; sudo update-rc.d z-way-server disable`.
```

Comment:
This isn't the case anymore?

Reply:
I agree that this should stay in the document.